coll/jq.go

```diff
@@ -0,0 +1,36 @@
+package coll
+
+import (
+	"github.com/itchyny/gojq"
+	"github.com/pkg/errors"
+)
+
+// JQ -
+func JQ(jqExpr string, in interface{}) (interface{}, error) {
+	query, err := gojq.Parse(jqExpr)
+	if err != nil {
+		return nil, errors.Wrapf(err, "couldn't parse JQ %s", jqExpr)
```

Comment:
I'd rather use `fmt.Errorf` than `pkg/errors` (which reminds me, I thought I got rid of uses of pkg/errors 🤔)

```suggestion
		return nil, fmt.Errorf("couldn't parse JQ %s: %w", jqExpr, err)
```